candidate.go

```diff
@@ -0,0 +1,131 @@
+package sdp
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/pions/webrtc/pkg/rtcerr"
+
+	"github.com/pkg/errors"
+)
+
+// Candidate is defined in https://tools.ietf.org/html/rfc5245.
+type Candidate struct {
```

Comment:
I guess this replaces the old `ICECandidate` structure -- how do I add a relatedaddress? i'm probably missing part of the spec

Reply:
Oh year. I ended up completely separating it from the ICECandidate. When you talked about relatedaddress are you talking about the server reflexive address generation?

In general lets keep this sdp library separate from the webrtc and self contained. That means if there is need for a copy of a similar struct in both, it is most beneficial to do that. We can just do conversions inside of webrtc.